.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  - push
+  - pull_request
```

Comment:
Do we really need `pull_request` when we have `push`?

Reply:
I think so.

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#push

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request